admin/src/app/app.component.html

```diff
@@ -1,4 +1,5 @@
 <div>
     <app-navbar></app-navbar>
+    <app-hotels-filter></app-hotels-filter>
```

Comment:
I'm just wondering is it expected that `app-hotels-filter` will presented on all pages of your app?

Reply:
At first I thought it will